SEBrowser/Scripts/TSX/Components/TrendData/Components/TrendMarkerTable.tsx

```diff
@@ -37,10 +39,11 @@ interface IProps {
 }
 
 const TrendMarkerTable = (props: IProps) => {
+    const dateTimeFormat = useSelector(SelectDateTimeFormat);
+
     const [trendMarkers, setTrendMarkers] = React.useState<TrendSearch.IMarker[]>([]);
     const [sortField, setSortField] = React.useState<string>('MeterName');
     const [ascending, setAscending] = React.useState<boolean>(true);
-    const momentFormat = "DD HH:mm:ss.SSS";
```

Comment:
This replacement doesn't work, moment format isn't the same as datetimeformat from settings.